videomaker/sass/blocks/_post-navigation-link.scss

```diff
@@ -1,10 +1,10 @@
 .wp-block-post-navigation-link {
 	flex-grow: 1;
+	border-top: 1px solid var(--wp--custom--color--primary);
```

Comment:
It looks like we get a border with no text (Next/Previous) on the first or last post when the border is on the container:

![image](https://user-images.githubusercontent.com/1645628/143275649-7a0f3723-76a5-418d-9ae6-e0d153a49a6d.png)

Reply:
aaaaah good catch, let me see how I can work around this

Reply:
`:empty` should catch it now!

Reply:
Yes awesome! Great idea, this works for me!